types/test/models/User.ts

```diff
@@ -71,22 +71,26 @@ User.init(
   }
 );
 
-User.afterSync(() => {
+User.hooks.add('afterSync', () => {
   sequelize.getQueryInterface().addIndex(User.tableName, {
-      fields: ['lastName'],
-      using: 'BTREE',
-      name: 'lastNameIdx',
-      concurrently: true,
+    fields: ['lastName'],
+    using: 'BTREE',
+    name: 'lastNameIdx',
+    concurrently: true,
   })
 })
 
 // Hooks
-User.afterFind((users, options) => {
+User.hooks.add('afterFind', (users: User[], options: FindOptions) => {
   console.log('found');
 });
 
+User.hooks.add('afterBulkCreate', (users: User[]) => {
+
+})
+
 // TODO: VSCode shows the typing being correctly narrowed but doesn't do it correctly
```

Comment:
is this still valid?

Reply:
Nah, but I will keep em as type assertions.